functors_task_1/student/include/impl.h

```diff
@@ -16,16 +22,18 @@ struct BeerOrganizer
  *
  * @note Only Corona and HoeGaarden are expensive
  */
-bool isExpensiveBeer(/**???*/)
+bool isExpensiveBeer(BeerBrand beer)
```

Comment:
_Mark "in" function params const by default, because modifying them is a bad practice_